daily-challenges/2024-11/29/problem.md

```
@@ -0,0 +1,9 @@
+# Playwright challenge
+Tập tục trang trí cây thông Noel bắt nguồn từ châu Âu, tượng trưng cho sự sống và hy vọng trong mùa đông giá lạnh. Cây thông thường được trang trí bằng đèn, quả cầu, và ngôi sao, tạo không khí ấm áp, vui tươi cho mùa lễ hội.
+
+# Đề bài:
```

Markdownlint (0.35.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)